src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_COST = 40;
+  const MANY_DAYS = 7;
+  const BIG_DISCOUNT = 50;
+  const FEW_DAYS = 3;
+  const SMALL_DISCOUNT = 20;
+
+  let totalCost = DAILY_COST * days;
+
+  if (days >= MANY_DAYS) {
+    return (totalCost = totalCost - BIG_DISCOUNT);
```

Comment:
```suggestion
    return totalCost - BIG_DISCOUNT;
```
You don't need to assign the value, just return it